Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,120 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background-color: #f0efef;
+  font-weight: 300px;
+  font-family: 'open Sans', sans-serif;
```

Comment:
Not a huge deal, since as you can see it worked fine, but when a font-family is named in title case it's probably safest to follow the way it's named explicitly, and 'Open Sans` is named with a capital 'O' and 'S' (https://www.google.com/fonts#UsePlace:use/Collection:Open+Sans) so probably best to not leave any room for error and follow that casing explicitly.